src/styles/components/sidebar.scss

```diff
@@ -1,3 +1,13 @@
+:root[data-theme="light"] {
+	--toggle-text-color: #828eb0;
+
+}
+
+:root[data-theme="dark"] {
+	--toggle-text-color: #39487a;
+}
+
+$toggle-text-color: var(--toggle-text-color);
```

Comment:
@Aringoaway Cannot this be loaded directly from asab-ui styles? I mean the toggle text color